.ci/configure_crio_for_kata.sh

```diff
@@ -12,25 +12,12 @@ set -o pipefail
 source /etc/os-release || source /usr/lib/os-release
 
 crio_config_file="/etc/crio/crio.conf"
-crio_config_dir="/etc/crio/crio.conf.d"
 runc_flag="\/usr\/local\/bin\/crio-runc"
 kata_flag="\/usr\/local\/bin\/containerd-shim-kata-v2"
 
-minor_crio_version=$(crio --version | head -1 | cut -d '.' -f2)
+minor_crio_version=$(crio --version | egrep -o "[0-9]+\.[0-9]+\.[0-9]+" | head -1 | cut -d '.' -f2)
 
-if [ "$minor_crio_version" -ge "18" ]; then
-	echo "Configure runtimes map for RuntimeClass feature with drop-in configs"
```

Comment:
@fidencio @haircommander I am removing this part as I found that when runc is configured in the main file (/etc/crio/crio.conf) and runc is the default runtime, and kata is configured in the drop-in file (/etc/crio/crio.conf.d/99-runtime.conf), the crio service cannot start. 
This is the error shown in the crio journal:
```
Oct 15 03:10:18 test01 crio[9358]: time="2020-10-15 03:10:18.940225732Z" level=fatal msg="Validating runtime config: default_runtime set to \"runc\", but no runtime entry was found for it"
```
This is what I have in the config files: 
```
cat /etc/crio/crio.conf.d/99-runtime.conf
[crio.runtime]
manage_ns_lifecycle = true
[crio.runtime.runtimes.kata]
runtime_path = "/usr/local/bin/containerd-shim-kata-v2"
runtime_root = "/run/vc"
runtime_type = "vm"
privileged_without_host_devices = true 
```

and on the main config file, the runc runtime is added and configured as default:

```
default_runtime = "runc"

...

[crio.runtime.runtimes.runc]
runtime_path = "/usr/local/bin/crio-runc"
runtime_type = "oci"
runtime_root = "/run/runc"
```

do you think this could be a bug or am I missing something?

Reply:
```
cat /etc/crio/crio.conf.d/99-runtime.conf
```
should read
```
[crio.runtime]
manage_ns_lifecycle = true
default_runtime = "kata"
[crio.runtime.runtimes.kata]
runtime_path = "/usr/local/bin/containerd-shim-kata-v2"
runtime_root = "/run/vc"
runtime_type = "vm"
privileged_without_host_devices = true 
```

it is possible it's a bug, but I don't understand why default_runtime isn't in 99-runtime.conf if it's clearly being added on line 26 below

Reply:
if we add kata as default runtime, like below in line 26, the error does not appear. But we still want to use runc as the default runtime and then specify kata through RuntimeClass k8s feature in our tests.

Reply:
ahhh so the runtime map doesn't append entries, but overwrites them. You need to manually include the default entry as well
```
[crio.runtime]
manage_ns_lifecycle = true
default_runtime = "kata"
[crio.runtime.runtimes.kata]
runtime_path = "/usr/local/bin/containerd-shim-kata-v2"
runtime_root = "/run/vc"
runtime_type = "vm"
privileged_without_host_devices = true 
[crio.runtime.runtimes.runc]
runtime_path = "/usr/local/bin/crio-runc"
runtime_type = "oci"
runtime_root = "/run/runc"
```

this was fixed in 1.19.0 in https://github.com/cri-o/cri-o/pull/3772

Reply:
ohh ok, didn't review issues on cri-o side, but thanks for the pointer.

Reply:
added runc to drop-in file.